New folder/lastmain.js

```diff
@@ -0,0 +1,103 @@
+// Ex1 returns an array with element elem repeated n times.
+const repeatElement = (element, repeat) => {
+  return Array(repeat).fill(element)
+}
+const repeatElementSecond = (element, repeat) => {
+  let newArray = []
+  for (let i = 0; i < repeat; i++) {
+    newArray = newArray.push(element)
+  }
+  return newArr
+}
+const repeatElementThird = (element, repeat) => {
+  let newArray = []
```

Comment:
`newArray` bên dưới ko bị gán lại, nên hãy đặt là `const`
```suggestion
  const newArray = []
```